src/__experimental__/components/radio-button/radio-button.d.ts

```diff
@@ -15,6 +15,8 @@ export interface RadioButtonProps extends CommonCheckableInputProps, MarginProps
   size?: OptionsHelper.SizesType;
   /** the value of the Radio Button, passed on form submit */
   value: string;
+  /** A callback to retrieve the input reference */
+  ref?: () => void;
```

Comment:
This isn't the correct way to add the ref in the TypeScript interface, see https://github.com/Sage/carbon/blob/5332e810e3515151efc0329f1130312610d6359c/src/components/accordion/accordion.d.ts#L54 or any other component using `forwardRef` for an example of how to do this

Reply:
It won't be a `HTMLDivElement` though, it will probably be a `HTMLInputElement` if the ref is passed down to the input itself

Reply:
Good point, changed, thanks